README.md

```diff
@@ -3,36 +3,115 @@
 ## About this project
 
 ### What is Nightfall SDK?
-Software Development Kit for interacting with Polygon Nightfall [about Polygon Nightfall](https://docs.polygon-nightfall.technology/).
+
+Software Development Kit for interacting with Polygon Nightfall.
+About [Polygon Nightfall](https://docs.polygon-nightfall.technology/).
 
 ### What is the SDK for?
 
-- Deposit your tokens from Ethereum L1 to Polygon Nightfall L2
-WIP
+- Deposit ERC20 tokens from Ethereum L1 to Polygon Nightfall L2
+- Transfer ERC20 token commitments on Polygon Nightfall L2
+- Withdraw ERC20 token commitments from Polygon Nightfall L2 to Ethereum L1
+- Check ERC20 balances on Polygon Nightfall L2
+- Safely export and import ERC20 transaction commitments from and to Polygon Nightfall L2
+- All the above applicable for ERC721 and ERC1155 token standards - WIP
```

Comment:
Let's leave out the import and the last bullet about other ERCs. We will add these when we put the features in master.

@luizoamorim I said to Stefan that the section is good, but moving forward I guess we'll have some kind of release notes that list the changes? E.g. new features, etc.

Thoughts?